src/components/Identity/index.js

```diff
@@ -10,30 +10,80 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-import { defer } from "../../utils";
+import { defer, assign } from "../../utils";
 import processIdSyncs from "./processIdSyncs";
+import createEvent from "../DataCollector/createEvent";
+import {
+  createHash,
+  validateCustomerIds,
+  normalizeCustomerIds,
+  bufferToHex
+} from "./util";
 
-const ECID_NAMESPACE = "ECID";
+import { COOKIE_NAMES } from "./constants";
+
+const { CUSTOMER_ID_HASH, EXPERIENCE_CLOUD_ID } = COOKIE_NAMES;
 
 const addIdsContext = (payload, ecid) => {
   // TODO: Add customer ids.
-  payload.addIdentity(ECID_NAMESPACE, {
+  payload.addIdentity(EXPERIENCE_CLOUD_ID, {
     id: ecid
   });
 };
 
 const createIdentity = ({ config, logger, cookie }) => {
   // We avoid reading the ECID from the cookie right away, because we
   // need to wait for the user to opt in first.
-  const getEcid = () => cookie.get(ECID_NAMESPACE);
+  const getEcid = () => cookie.get(EXPERIENCE_CLOUD_ID);
   let optIn;
   let responseRequested = false;
   let deferredForEcid;
+  let network;
+  let lifecycle;
+  const customerIds = {};
```

Comment:
To make sure we don't lose @Aaronius 's comment about the name:

Let's add a TODO here to discuss renaming `customer` to `user` with Product. It might be too late because it's a known construct within Adobe and our customers, but let's try for sure.

@jileeshs

Reply:
> Hey @jileeshs, are you done making changes / force pushing, so we can start reviewing?

@jfkhoury ,
Yes I am done making changes. 
@jfkhoury , @Aaronius , @jonsnyder  please review

Reply:
> To make sure we don't lose @Aaronius 's comment about the name:
> 
> Let's add a TODO here to discuss renaming `customer` to `user` with Product. It might be too late because it's a known construct within Adobe and our customers, but let's try for sure.
> 
> @jileeshs

Added a TODO commend near the exposed command names

Reply:
@jonsnyder made an interesting comment [here](https://github.com/adobe/alloy/pull/114#discussion_r309428045) that was from a different perspective than mine. When approached from his perspective, "customer" seems sufficiently appropriate.